.eslintignore

```diff
@@ -1,5 +1,12 @@
-.circleci/*
-.githooks/*
-.idea/*
-.vscode/*
-node_modules/*
+.circleci
+.githooks
+.idea
+.vscode
+.yarn*
+dist
+node_modules
+*.d.ts
+*.js
+
+# This has a separate self-contained .eslintrc.js configuration
+documentation/samples
```

Comment:
There is a separate config for these files, but if you add this line then those files aren't linted at all. In the main branch, running `make lint` catches errors in those files, using the local config. This should be removed.

Reply:
Indeed -- I'll remove this as it was added in error along with a `root` directive in the lower level eslint config.